exporter/mezmoexporter/README.md

```diff
@@ -0,0 +1,30 @@
+# Mezmo Exporter
+
+This exporter supports sending OpenTelemetry log data to [Mezmo](https://mezmo.com).
+
+# Configuration options:
+
+- `endpoint` (required): LogService's [Region](https://cloud.tencent.com/document/product/614/56473).
```

Comment:
Looking at the examples and default values, I don't think the endpoint is the same as the region?

Reply:
`endpoint` should have been `ingest_url`.   Corrected.